scanner/src/main/kotlin/scanners/scancode/ScanCode.kt

```diff
@@ -124,6 +127,8 @@ class ScanCode internal constructor(
     override fun command(workingDir: File?) =
         listOfNotNull(workingDir, if (Os.isWindows) "scancode.bat" else "scancode").joinToString(File.separator)
 
+    override fun getVersionRequirement(): RangesList = RangesListFactory.create("[3.0.0,)")
```

Comment:
Nit: We could simplify this to `">=3.0.0"`, but that's true for several existing ranges as well.

Reply:
I'll make a follow-up PR.

Reply:
@sschuberth see https://github.com/oss-review-toolkit/ort/pull/6673